routes/todo.go

```diff
@@ -0,0 +1,55 @@
+package routes
+
+import (
+	"fmt"
+	"github.com/gofiber/fiber/v2"
+	"github.com/narvikd/errorskit"
+	"github.com/narvikd/fiberparser"
+	"strconv"
+)
+
+func (h handler) GetAll(c *fiber.Ctx) error {
+	todoList := h.DB.FetchAll()
+
+	if len(todoList) == 0 {
+		return FailResponse("There are no TODOs to be returned", c, fiber.StatusNotFound)
```

Comment:
Through your app you're printing and returning errors with capital letters, please standardise those to minus.
I will mark them for you.